apps/storybook/src/AxialSelectToZoom.stories.tsx

```diff
@@ -28,7 +29,7 @@ const Template: Story<AxialSelectToZoomProps> = (args) => {
       abscissaConfig={{ visDomain: [0, oneD.length], showGrid: true }}
       ordinateConfig={{ visDomain: domain, showGrid: true }}
     >
-      <Pan />
+      <Pan modifierKey={modifierKey?.length === 0 ? 'Control' : undefined} />
```

Comment:
In a few stories, the pan interaction was conflicting with the selection interaction. So I know set the pan's `modifierKey` dynamically so it never conflicts.